.golangci.yml

```diff
@@ -0,0 +1,31 @@
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 10m
+
+  skip-dirs:
+    - testdata$
+    - test/mock
+
+  skip-files:
+    - ".*\\.pb\\.go"
```

Comment:
is there any proto on controller manager?

Reply:
removed